src/app.tsx

```diff
@@ -42,7 +42,7 @@ if (dsn) {
 // Create redux store with history
 const initialState = Immutable.Map();
 const store = configureStore(initialState, history);
-const MOUNT_NODE = document.getElementById('app');
+const MOUNT_NODE = document.getElementById('app')!;
```

Comment:
What does that exclamation mark do?

Reply:
ensures the result is treated as not `null` or `undifined`